.github/workflows/tests.yaml

```
@@ -33,10 +33,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        python-version: [3.6, 3.7, 3.8, 3.9]
-        exclude:
-        - os: windows-latest
-          python-version: 3.6
+        python-version: [3.7, 3.8, 3.9]
```

Test on Python 3.10 too?
```suggestion
        python-version: ['3.7', '3.8', '3.9', '3.10']
```